From a8648dbdbc213401cf9bdb5c9ab9de562ed5b7d0 Mon Sep 17 00:00:00 2001
From: Peter Jung <admin@ptr1337.dev>
Date: Sat, 31 Jan 2026 13:14:34 +0100
Subject: [PATCH 068/100] Revert "drm/amd/display: Save HDMI gaming info to
 edid caps"

This reverts commit ed0b09cf25ee2de29ea9801879baff647e499902.
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_helpers.c | 7 +------
 drivers/gpu/drm/amd/display/dc/dc_types.h                 | 7 +------
 2 files changed, 2 insertions(+), 12 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_helpers.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_helpers.c
index c6efca2ad..e03321c22 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_helpers.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_helpers.c
@@ -136,12 +136,7 @@ enum dc_edid_status dm_helpers_parse_edid_caps(
 				  edid_caps->display_name,
 				  AUDIO_INFO_DISPLAY_NAME_SIZE_IN_CHARS);
 
-	if (connector->display_info.is_hdmi) {
-		edid_caps->edid_hdmi = true;
-		edid_caps->allm = connector->display_info.hdmi.allm;
-		edid_caps->fva = connector->display_info.hdmi.vrr_cap.fva;
-		edid_caps->hdmi_vrr = connector->display_info.hdmi.vrr_cap.supported;
-	}
+	edid_caps->edid_hdmi = connector->display_info.is_hdmi;
 
 	apply_edid_quirks(dev, edid_buf, edid_caps);
 
diff --git a/drivers/gpu/drm/amd/display/dc/dc_types.h b/drivers/gpu/drm/amd/display/dc/dc_types.h
index 012691adf..b5aa03a3e 100644
--- a/drivers/gpu/drm/amd/display/dc/dc_types.h
+++ b/drivers/gpu/drm/amd/display/dc/dc_types.h
@@ -206,14 +206,9 @@ struct dc_edid_caps {
 
 	uint32_t max_tmds_clk_mhz;
 
-	/* HDMI 2.0 caps */
+	/*HDMI 2.0 caps*/
 	bool lte_340mcsc_scramble;
 
-	/* HDMI 2.1 caps */
-	bool allm;
-	bool fva;
-	bool hdmi_vrr;
-
 	bool edid_hdmi;
 	bool hdr_supported;
 	bool rr_capable;
-- 
2.52.0

